Library/Homebrew/deprecate_disable.rb

```diff
@@ -75,6 +75,14 @@ def message(formula_or_cask)
       end
     end
 
+    replacement = if formula_or_cask.deprecated?
+      formula_or_cask.deprecation_replacement
+    elsif formula_or_cask.disabled?
+      formula_or_cask.disable_replacement
+    end
+
+    message += "\nConsider replacing it with:\n    brew install #{replacement}" if replacement.present?
```

Comment:
```suggestion
    message += "\nReplacement:\n    brew install #{replacement}" if replacement.present?
```

"Consider replacing" is both too long and too "soft"; if we feel the need to hedge a replacement, we shouldn't make it.

Also: can this be adjusted to use `<<~EOS` instead of embedded newlines and two spaces instead of 4 for the "indentation"? Thanks!